:

```diff
@@ -38,6 +83,7 @@ def list_content(
         url = f"{self.base_url}/repos/{org}/{repo}/contents/{folder}?ref={branch}"
         response = self.client.get(url)
         response.raise_for_status()
+        print(response.json())
```

Comment:
You probably do not want this print statement here

Reply:
Hmm this file should even exists ... I don't quite understand why it is visible here (and not in the folder in this branch).